src/iqstat.cpp

```diff
@@ -208,12 +208,15 @@ Print the main usage/help information.
 void usage() {
     char *msgs[] = {
         "Usage: iqstat [-luvVh] [-u user] [ruleId]",
-        "Show information about your pending iRODS rule executions",
-        "or for the entered user.",
-        " -a        display requests of all users",
-        " -l        for long format",
-        " -u user   for the specified user",
-        " ruleId for the specified rule (implies long format)",
+        "Show information about pending iRODS rule executions (i.e. delayed rules).",
+        "Listings are produced via a general query of the R_RULE_EXEC table.  By default,",
```

Comment:
Let's remove the first statement here. The reader doesn't need to know how the information is gathered.

Keep the statement about the default behavior though.

Reply:
done!